File: app/src/main/java/com/jakubwyc/mysecretvault/view/VaultView.java
package com.jakubwyc.mysecretvault.view;

import com.jakubwyc.mysecretvault.model.Treasure;

import java.util.List;

public interface VaultView {

    void goToEditTreasureScreen(Treasure treasure);

    void showSnackbar(int messageId);

    void renderTreasures(List<Treasure> treasures);

}

File: app/src/main/java/com/jakubwyc/mysecretvault/vault/VaultView.kt
package com.jakubwyc.mysecretvault.vault

import com.jakubwyc.mysecretvault.model.Treasure

interface VaultView {

    fun goToEditTreasureScreen(treasure: Treasure)

    fun showSnackbar(messageId: Int)

    fun renderTreasures(treasures: List<Treasure>)

}
